apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31011&commitId=eb61eb7b49fcfbb4ddf48986ee0d544e425e5b41&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Menu Example Divider Horizontal.chromium.png|76| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>